conjure-java-core/src/integrationInput/java/com/palantir/product/SingleUnion.java

```diff
@@ -87,24 +87,29 @@ public String toString() {
 
         private Function<String, T> unknownVisitor;
 
+        @Override
         public UnknownStageVisitorBuilder<T> foo(Function<String, T> fooVisitor) {
             Preconditions.checkNotNull(fooVisitor, "fooVisitor cannot be null");
             this.fooVisitor = fooVisitor;
             return this;
         }
 
+        @Override
         public CompletedStageVisitorBuilder<T> unknown(Function<String, T> unknownVisitor) {
             Preconditions.checkNotNull(unknownVisitor, "unknownVisitor cannot be null");
             this.unknownVisitor = unknownVisitor;
```

Comment:
not necessarily for this PR, we can update this to the form:
```java
this.unknownVisitor = Preconditions.checkNotNull(unknownVisitor, "unknownVisitor cannot be null");
```

Reply:
Yes. This struck me too. Can't say I'm a fan of that pattern, but it could very well be that I'm just not used to it.